docs/src/userguide/loading_iris_cubes.rst

```diff
@@ -215,6 +215,14 @@ constraint to ``load``::
     level_10 = iris.Constraint(model_level_number=10)
     cubes = iris.load(filename, forecast_6 & level_10)
 
+.. note::
+
+    Whilst ``&`` is supported, the ``|`` that might reasonably be expected
+    is not. This is because "or" constraints could lead to a cube which did
+    not cover a hyper-rectangular region if these constraints were on
+    different coordinates. If an "or" type constraint on a single coordinate
+    is useful then it may be obtained by passing a function to :class:`iris.Constraint`.
```

Comment:
@wjbenfold Lovely contribution, thanks, but let's separate concerns here.

I'd prefer that you remove this update to the docs from this PR and include it in a separate PR.

Is that okay?

Reply:
Huh, I didn't think I'd added this file and I can't see from my command history how it got in here. I'll remove it